orc8r/cloud/helm/orc8r/templates/controller.deployment.yaml

```diff
@@ -115,15 +117,21 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: status.podIP
+            - name: SERVICE_REGISTRY_MODE
+              value: {{ .Values.controller.spec.service_registry.mode }}
+            - name: HELM_RELEASE_NAME
+              value: {{ .Release.Name }}
           livenessProbe:
             tcpSocket:
               port: 9081
             initialDelaySeconds: 10
             periodSeconds: 30
-          readinessProbe:
-            tcpSocket:
-              port: 9081
-            initialDelaySeconds: 5
-            periodSeconds: 10
+          # Readiness probe prevents service registry mode 'k8s' from working
+          # properly
+          #readinessProbe:
+          #  tcpSocket:
+          #    port: 9081
+          #  initialDelaySeconds: 5
+          #  periodSeconds: 10
```

Comment:
Why is this?

Reply:
We have a cylic dependency right now. Obsidian depends on the service registry before starting up to get reverse proxy paths. The service registry service won't backend to the controller pod correctly until the readiness probe passes.

We could instead start the obsidian server without any proxy paths and just wait for the background go routine to update the paths after 30s, but this should go away once the controller pod is split anyways